01_git/Makefile

```diff
@@ -0,0 +1,12 @@
+CC=gcc
+CC_FLAGS=-std=c99 -Wall
+
+CC_SOURCES=main.c
+
+rock_paper_scissors:
+	$(CC) $(CC_FLAGS) $(CC_SOURCES) -o$@
+
+all: rock_paper_scissors
+
+clean:
+	rm rock_paper_scissors
```

Comment:
Add newline at the end of the file

Reply:
Think about automatic cleaning the whole directory from build artifacts

Reply:
Automatic cleaning the whole directory from build artifacts was added.

Reply:
Newline at the end of the file was added.